discovery.md

```diff
@@ -538,6 +539,15 @@ This MUST return an array of zero or more Services. The array MUST contain all
 Services available via this Discovery Enpoint. Any Service previously returned
 to a client that does not appear in this result can be assumed to be deleted.
 
+The collection of services MAY be filtered by supplying an attribute as a query
+parameter.  Doing so will cause only Services with an exact match for that
+attribute to be included in the result.
+
+Discovery endpoints MUST support filtering with the following attributes and
+MAY ignore other filters.
```

Comment:
I think I'd prefer to generate an error if they don't support the specified filter - otherwise there's no way for the client to know if the result set is empty, or the filter was ignored.

Reply:
My first instinct is to agree with that being reasonable.  And then I go down an OPTIONS like rabbit hole for tools to identify which filters are supported.  I suppose that's not the end of the world and services could do binary discoveries on which filters are supported ... but do you have a thought on that?  Others welcome to express opinions too of course :D